mlprimitives/jsons/sklearn.linear_model.Lasso.json

```diff
@@ -0,0 +1,129 @@
+                "default": true
+            },
+            "max_iter": {
+                "type": "int",
+                "description": "The maximum number of iterations",
+                "default": 1000
+            },
+            "warm_start": {
+                "type": "bool",
+                "description": "When set to True, reuse solution of the previous call to fit as initialization, otherwise, erase the previous solution.",
+                "default": false
+            },
+            "random_state": {
+                "type": "multitype",
+                "description": "The seed of the pseudo random number generator to use when shuffling the data.",
+                "types": {
+                    "int":{
+                        "description": "random_state is the seed used by the random number generator"
+                    },
+                    "RandomState":{
+                        "description": "If an array is passed, penalties are assumed to be specific to the targets. Hence they must correspond in number."
+                    },
+                    "NoneType": {
+                        "description": "If none, the random number generator is the RandomState instance used by np.random"
+                    }
+                },
+                "default": null
+            },
+            "precompute": {
+                "type": "multitype",
+                "description": "Whether to use a precomputed Gram matrix to speed up calculations. Gram natrix can also be passed as argument.",
+                "types":{
+                    "bool":{
+                        "description": "whether to use the precomputed gram matrix"
+                    },
+                    "array":{
+                        "description": "Gram matrix to speed up calculations"
+                    }
+                },
+                "default": false
+            },
+            "selection": {
+                "type": "str",
+                "description": "If set to ‘random’, a random coefficient is updated every iteration rather than looping over features sequentially by default.",
+                "values": ["cyclic", "random"],
+                "default": "cyclic"
+            }
+        },
+        "tunable": {
+            "normalize": {
+                "type": "bool",
+                "default": false,
+                "description": "This parameter is ignored when fit_intercept is set to False. If True, the regressors X will be normalized before regression by subtracting the mean and dividing by the l2-norm."
+            },
+            "fit_intercept": {
+                "type": "bool",
+                "default": true,
+                "description" : "Whether to calculate the intercept for this model. If set to False, no intercept will be used in calculations (e.g. data is expected to be already centered)."
+            },
+            "alpha": {
+                "type": "multitype",
```

Comment:
Multitype hyperparameters are not supported yet by BTB, so for now let's use only basic types for the tunables.
On the other side, `array` is not a valid type for a hyperparameter, so I suggest removing this branch and making all this hyperparameter a `float`.

Also remember to add a valid range (see comment below)